src/main/java/commission/dao/PersonDaoImpl.java

```diff
@@ -0,0 +1,61 @@
+package commission.dao;
+
+import commission.entity.Person;
+import commission.mapper.PeopleRowMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class PersonDaoImpl implements SalesCommissionDao<Person>{
+
+    private final JdbcTemplate jdbcTemplate;
+    private final PeopleRowMapper rowMapper = new PeopleRowMapper();
+
+    @Autowired
+    public PersonDaoImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public List<Person> findAll() {
+        var query = "SELECT * FROM people";
```

Comment:
Missed this initially, try  not to use `*` and rather define the columns you want to select explicitly. This is good practice as it means you only return the data you care about, some tables might have more rows than you need for something in the application.

In many systems the database is often deployed on a different server / virtual machine than the application, so all columns and data retrieved have to be sent over the network. You always want to be as efficient as you can with  network calls and they add overhead.

Reply:
Thanks. Makes sense. It was on my to-do list too.